docs/src/.vitepress/config.mts

```diff
@@ -73,6 +74,7 @@ export default defineConfig({
       { text: 'Methods',
         items: [
           { text: 'Overview', link: '/methods' },
+          { text: 'resample', link: '/resample_proj'},
```

Comment:
This should be capitalized IMO, or we could call the page `Reprojection` (since that's what it does - resampling to me implies a simple down- or up-sampling without nonlinearity).

Reply:
maybe both? `Reprojection and resample`.

Reply:
```suggestion
          { text: 'Reprojection and resampling', link: '/resample_proj'},